public/javascripts/SVLabel/build/SVLabel.js

```diff
@@ -11735,39 +11747,207 @@ function ModalMissionComplete ($, d3, L) {
+        map.on("viewreset", reset);
+
+        // this puts stuff on the map! 
+        reset();
+        transition();
+
+        // Reposition the SVG to cover the features.
+        function reset() {
+            var bounds = d3path.bounds(collection),
+                topLeft = bounds[0],
+                bottomRight = bounds[1];
+
+            // Setting the size and location of the overall SVG container
+            svg.attr("width", bottomRight[0] - topLeft[0] + 120)
+                .attr("height", bottomRight[1] - topLeft[1] + 120)
+                .style("left", topLeft[0] - 50 + "px")
+                .style("top", topLeft[1] - 50 + "px");
+
+            // set opacity to zero before it is rendered
+            linePath.style("opacity", "0").attr("d", toLine);
+            g.attr("transform", "translate(" + (-topLeft[0] + 50) + "," + (-topLeft[1] + 50) + ")");
+
+        } // end reset
+
+        
+        function transition() {
+            linePath.transition()
+                .duration(3000)
+                .attrTween("stroke-dasharray", tweenDash)
+                .each("end", function() {
+                    if(index < max){
+                        // recursively call the next animation render when this one is done
+                        _animateMissionTasks(coll, index+1, max);
+                    }
+                    else{
+                        // unlock zooming
+                        map.touchZoom.enable();
+                        map.doubleClickZoom.enable();
+                        map.scrollWheelZoom.enable();
+                        map.boxZoom.enable();
+                        //render the complete path as plain svg to avoid scaling issues
+                        renderPath(coll);
+
+                    }
+                    
+                }); 
+        } //end transition
+
+        // this function feeds the attrTween operator above with the 
+        // stroke and dash lengths
+        function tweenDash() {
+            return function(t) {
+                // reapply opacity to prevent phantom segment
+                linePath.style("opacity", "1");
+                //total length of path (single value)
+                var l = linePath.node().getTotalLength(); 
+                interpolate = d3.interpolateString("0," + l, l + "," + l);
+                var p = linePath.node().getPointAtLength(t * l);
+
+                return interpolate(t);
+            }
+        } //end tweenDash
+
+        function projectPoint(x, y) {
+            var point = map.latLngToLayerPoint(new L.LatLng(y, x));
+            this.stream.point(point.x, point.y);
+        }
+
+        function applyLatLngToLayer(d) {
+            var y = d.geometry.coordinates[1];
+            var x = d.geometry.coordinates[0];
+            return map.latLngToLayerPoint(new L.LatLng(y, x));
+        }
+    }
+
+    // create array of street edge ids from mission tasks
+    function streetEdgeIds(missionTasks){
```

Comment:
Why do we need a method here? Again, isn't it a one liner? Like, `missionTasks.map( function (t) { return t.getStreetEdgeId(); })`